omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -20,6 +20,10 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
      * @type {Function}
      */
     self.saveCondition = self.saveCondition || function () {
+        if($scope.condition.onSetDate>$scope.condition.endDate){
```

Comment:
Can you put some spaces around the > sign?

Reply:
Sure :)